integration/tests/cook/test_cli.py

```diff
@@ -1748,7 +1748,15 @@ def test_usage(self):
         self.assertEqual(0, cp.returncode, cp.stderr)
         self.assertEqual(3, len(uuids))
         uuid_7, uuid_8, uuid_9 = uuids
-        all_uuids = [uuid_1, uuid_2, uuid_3, uuid_4, uuid_5, uuid_6, uuid_7, uuid_8, uuid_9]
+
+        cp, uuids = cli.submit(command, self.cook_url, submit_flags='--cpus 0.2 --mem 32 --pool alpha')
```

Comment:
We can't assume here that there's a pool named alpha. We want these tests to work on clusters that don't have pools too. Or that have pools with different names. Search for calls to `util.all_pools` or `util.active_pools` to see examples of existing tests that check whether or not there are pools in the target cluster.